crypto/mem.c

```diff
@@ -136,16 +136,17 @@ static void parseit(void)
  * Some rand() implementations aren't good, but we're not
  * dealing with secure randomness here.
  */
-#ifdef _WIN32
-# define random() rand()
-#endif
+# ifdef _WIN32
+#  define random() rand()
+# endif
 /*
  * See if the current malloc should fail.
  */
 static int shouldfail(void)
 {
     int roll = (int)(random() % 100);
     int shoulditfail = roll < md_fail_percent;
+# ifndef _WIN32
```

Comment:
It would be more than appropriate to drop a line of commentary saying that the code segment is suppressed on Windows because POSIX-like file descriptors are non-inheritable.

Reply:
I've just added such a line.